src/components/header/Header.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Header() {
+  return <div>Header</div>;
+}
```

Comment:
Be sure to add in a total count of the shopping list items within the header component.